src/Ardalis.Extensions/Math/Add.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace Ardalis.Extensions.Math
+{
+    // Naming: https://en.wikipedia.org/wiki/Template:Arithmetic_operations
+    public static partial class MathExtensions
+    {
+        public static int Add(this short augend, short addend)
```

Comment:
Should this return a `short`? I know it's possible it could overflow and require an `int` but following that logic, your `Add(int, int)` method should return `long` (it returns `int`). We should be consistent.

Reply:
I chose int as the return type here due to the fact that the addition operator is only defined on the int, uint, long, and ulong types. The sbyte, byte, short, ushort, and char types are implicitly converted to type int and as a result return int.
I wanted to stick with standard functionality, ie `x + y` performs the same operation as `x.Add(y)`
See: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/arithmetic-operators
And: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/expressions#numeric-promotions